package.json

```diff
@@ -1,16 +1,45 @@
 {
   "name": "fold",
-  "version": "0.12.0",
+  "version": "0.13.0",
   "description": "FOLD file format for origami models, crease patterns, etc.",
-  "main": "lib/index.js",
+  "main": "dist/index.civet.js",
```

Comment:
With the way the current unplugin works, it would break all the imports. Though we could maybe the file matcher for transpiled civet in it from "[path].civet.js" to "civet:[path].js", in which case the final extension should just be js. I'm not sure if it would make much of a difference though